src/main.rs

```diff
@@ -220,6 +270,7 @@ fn test_read_canister_ids_from_file() {
         governance_canister_id: CanisterId::from_str("rrkah-fqaaa-aaaaa-aaaaq-cai").unwrap(),
         ledger_canister_id: CanisterId::from_str("ryjl3-tyaaa-aaaaa-aaaba-cai").unwrap(),
         root_canister_id: CanisterId::from_str("r7inp-6aaaa-aaaaa-aaabq-cai").unwrap(),
+        dapp_canister_id_list: vec![],
```

Comment:
Can you test all cases for specifying dapp_canister_id_list, not just the empty case?